src/XML/src/FMI3/fmi3_xml_model_description.c

```diff
@@ -92,10 +92,8 @@ fmi3_xml_model_description_t * fmi3_xml_allocate_model_description( jm_callbacks
     fmi3_xml_init_type_definitions(&md->typeDefinitions, cb);
 
     jm_vector_init(jm_named_ptr)(&md->variablesByName, 0, cb);
-
-    md->variablesOrigOrder = 0;
-
-    md->variablesByVR = 0;
+    jm_vector_init(jm_voidp)(&md->variablesOrigOrder, 0, cb);
+    jm_vector_init(jm_voidp)(&md->variablesByVR, 0, cb);
```

Comment:
Refactor: I saw no reason to have them being pointer, so made them non-pointer members